docs/api/alerting/update_rule.asciidoc

```diff
@@ -23,91 +32,117 @@ Update the attributes for an existing rule.
 updated the rule, you might change its behavior.
+* Though some properties are optional, when you update the rule the existing
+property values are overwritten with default values. Therefore, it is
+recommended to explicitly set all property values.
 ====
 
 [[update-rule-api-path-params]]
-==== Path parameters
+=== {api-path-parms-title}
 
 `id`::
-  (Required, string) The ID of the rule that you want to update.
+(Required, string) The ID of the rule that you want to update.
 
 `space_id`::
-  (Optional, string) An identifier for the space. If `space_id` is not provided in the URL, the default space is used.
+(Optional, string) An identifier for the space. If `space_id` is not provided in
+the URL, the default space is used.
 
+[role="child_attributes"]
 [[update-rule-api-request-body]]
-==== Request body
-
-`name`::
-  (Required, string) A name to reference and search.
-
-`tags`::
-  (Optional, string array) A list of keywords to reference and search.
+=== {api-request-body-title}
 
-`schedule`::
-  (Required, object) When to run this rule. Use one of the available schedule formats.
+`actions`::
+(Optional, object array) An array of action objects. The default value is an
+empty array (`[]`).
 +
-._Schedule Formats_.
+.Properties of the action objects:
 [%collapsible%open]
 =====
-A schedule uses a key: value format.  {kib} currently supports the _Interval format_ , which specifies the interval in seconds, minutes, hours, or days at which to execute the rule.
+`id`:::
+(Required, string) The identifier of the action.
 
-Example: `{ interval: "10s" }`, `{ interval: "5m" }`, `{ interval: "1h" }`, `{ interval: "1d" }`.
+`group`:::
+(Required, string) Grouping actions is recommended for escalations for different
+types of alerts. If you don't need this, set the value to `default`.
 
+`params`:::
+(Required, object) The map to the `params` that the
+<<action-types,connector type>> will receive. The `params` are handled as
+Mustache templates and passed a default set of context.
 =====
 
-`throttle`::
-  (Optional, string) How often this rule should fire the same actions. This will prevent the rule from sending out the same notification over and over. For example, if a rule with a `schedule` of 1 minute stays in a triggered state for 90 minutes, setting a `throttle` of `10m` or `1h` will prevent it from sending 90 notifications during this period.
+`name`::
+(Required, string) A name to reference and search.
 
 `notify_when`::
-  (Required, string) The condition for throttling the notification: `onActionGroupChange`, `onActiveAlert`,  or `onThrottleInterval`.
+(Required, string) The condition for throttling the notification:
+`onActionGroupChange`, `onActiveAlert`,  or `onThrottleInterval`.
 
 `params`::
-  (Required, object) The parameters to pass to the rule type executor `params` value. This will also validate against the rule type params validator, if defined.
+(Required, object) The parameters to pass to the rule type executor `params`
```

Comment:
Is "executor" or a replacement word needed here? Or can we just drop it?
```suggestion
(Required, object) The parameters to pass to the rule type `params`
```

Reply:
I think we can leave it out.